docs/application/dotnet/guides/user-interface/nui/relative-layout.md

```diff
@@ -73,14 +82,533 @@ The `HorizontalAlignment` and the `VerticalAlignment` properties describe how to
+- `RelativeLayout.TopRelativeOffset="0.0"` - view's top is at the yellowView's top.
+- `RelativeLayout.BottomRelativeOffset="0.2"` - view's bottom is at the position of 0.2 width from the yellowView's left.
+- `RelativeLayout.FillHorizontal="true"` and `RelativeLayout.FillVertical="true"` - Since the view does not have its Width and Height,the view should be filled the relative position(left, right, top, bottom).
+
+</span>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+    <div id="RelativeLayout-GreenView-Xaml" class="tabcontent">
+        <table>
+            <tbody>
+                <tr>
+<span style="display:block">
+
+```xaml
+<View
+    x:Name="greenView"
+    BackgroundColor="Green"
+    RelativeLayout.LeftTarget="{x:Reference Name=yellowView}"
+    RelativeLayout.RightTarget="{x:Reference Name=yellowView}"
+    RelativeLayout.TopTarget="{x:Reference Name=yellowView}"
+    RelativeLayout.BottomTarget="{x:Reference Name=yellowView}"
+
+    RelativeLayout.LeftRelativeOffset="0.0"
+    RelativeLayout.RightRelativeOffset="0.2"
+    RelativeLayout.TopRelativeOffset="0.0"
+    RelativeLayout.BottomRelativeOffset="0.2"
+
+    RelativeLayout.FillHorizontal="True"
+    RelativeLayout.FillVertical="True"/>
+```
+
+</span>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</div>
+
+### Black view
+
+The following code shows how to create black view related to yellowView and greenView.
+
+![RelativeLayoutExampleBlackView](./media/relative_layout_example_black_view.png)
+
+<div id="TabSection6">
+    <div class="sampletab " id="ProjectCreateTab">
+        <button id="RelativeLayout-BlackView-CSharp" class="tablinks " onclick="openTabSection(event, 'RelativeLayout-BlackView-CSharp', 'TabSection6') ">C#</button>
+        <button id="RelativeLayout-BlackView-Xaml" class="tablinks " onclick="openTabSection(event, 'RelativeLayout-BlackView-Xaml', 'TabSection6') ">Xaml</button>
+    </div>
+    <div id="RelativeLayout-BlackView-CSharp" class="tabcontent">
+        <table>
+            <tbody>
+                <tr>
+<span style="display:block">
+
+- View's left and top targets are set to `greenView`.
+- View's right and bottom targets are set to `yellowView`.
+- `RelativeLayout.SetLeftRelativeOffset(blackView, 1.0f)` - view's left is on the greenView's right.
+- `RelativeLayout.SetRightRelativeOffset(blackView, 1.0f)` - view's right is on the yellowView's right.
+- `RelativeLayout.SetTopRelativeOffset=(blackView, 1.0f)` - view's top is on the greenView's bottom.
+- `RelativeLayout.SetBottomRelativeOffset=(blackView, 1.0f)` - view's bottom is on the yellowView's bottom.
+- `RelativeLayout.SetSetFillHorizontal=(blackView, true)` and `SetRelativeLayout.FillVertical(blackView, true)` - Since the view does not have its Width and Height,the view should be filled the relative position(left, right, top, bottom).
```

Comment:
```suggestion
- `RelativeLayout.SetFillHorizontal(blackView, true)` and `RelativeLayout.SetFillVertical(blackView, true)` - Since the view does not have its Width and Height, the view should be filled the relative position(left, right, top, bottom).
```